src/matter/cluster/GeneralCommissioningCluster.ts

```diff
@@ -45,18 +46,26 @@ const SetRegulatoryConfigRequestT = ObjectT({
     breadcrumbStep: Field(2, UnsignedIntT),
 });
 
+/**
+ * This cluster is used to manage global aspects of the Commissioning flow.
+ */
 export const GeneralCommissioningClusterSpec = ClusterSpec(
     0x30,
     "General Commissioning",
     {
-        breadcrumb: AttributeSpec(0, UnsignedIntT),
+        breadcrumb: AttributeSpec(0, UnsignedIntT), // INT64U ??
         commissioningInfo: AttributeSpec(1, BasicCommissioningInfoT),
         regulatoryConfig: AttributeSpec(2, RegulatoryLocationTypeT),
         locationCapability: AttributeSpec(3, RegulatoryLocationTypeT),
+        supportsConcurrentConnections: AttributeSpec(4, BooleanT, true),
     },
     {
-        armFailSafe: CommandSpec(0, ArmFailSafeRequestT, 1, SuccessFailureReponseT),
-        updateRegulatoryConfig: CommandSpec(2, SetRegulatoryConfigRequestT, 3, SuccessFailureReponseT),
-        commissioningComplete: CommandSpec(4, NoArgumentsT, 5, SuccessFailureReponseT),
+        // Arm the persistent fail-safe timer with an expiry time of now + ExpiryLengthSeconds using device clock
```

Comment:
Those comments should be in the /** */ format.

And yes! Visual Studio magically displays those comments when accessing the method on the generated types!
I just tried.

So ClusterClient(interactionClient, 0, GeneralCommissioningClusterSpec).armFailSafe will display this comment...
And it is also displayed on the implementation of the Cluster on the device side.

Same for the comments on attributes, Visual Studio correctly displays them on generated types.